napari_aicssegmentation/core/layer_reader.py

```diff
@@ -31,3 +31,24 @@ def get_channels(self, layer: Layer) -> List[Channel]:
         for index in range(img.shape[index_c]):
             channels.append(Channel(index))
         return channels
+
+    def get_channel(self, index: int):
+        """
+            Get the channel at this index
+        """
+        return self.get_channels()[index]
+
+    def get_selected_channel(self, channel: Channel, layer: Layer, channel_index: int = 0):
```

Comment:
This is good. Let's maybe call this `get_channel_data` and / or at least annotate the return as a `numpy.ndarray` to make it clear we're returning the data and not just the channel information.